.github/workflows/docker.yml

```diff
@@ -0,0 +1,35 @@
+name: Build & push container image
```

Comment:
Can you change also the filename please? I am pre-approving, ready to go!

Reply:
done!